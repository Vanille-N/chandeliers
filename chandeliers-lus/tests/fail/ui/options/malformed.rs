chandeliers_lus::decl! {
    #[mai]
    node system() returns ();
    let tel
}

fn main() {}
