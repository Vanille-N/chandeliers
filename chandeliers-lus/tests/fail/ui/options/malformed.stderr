error: Unknown or malformed attribute "mai"
 --> tests/fail/ui/options/malformed.rs:2:5
  |
2 |     #[mai]
  |     ^^^^^^
